STEER/AOD/AliAODEvent.cxx

```diff
@@ -733,17 +733,46 @@ void AliAODEvent::ReadFromTree(TTree *tree, Option_t* opt /*= ""*/)
     // Try to find AliAODEvent
   AliAODEvent *aodEvent = 0;
   aodEvent = (AliAODEvent*)tree->GetTree()->GetUserInfo()->FindObject("AliAODEvent");
+  
+  TList* friendL;
   if(aodEvent){
     // This event is connected to the tree by definition, just say so
     aodEvent->SetConnected();
       // Check if already connected to tree
     TList* connectedList = (TList*) (tree->GetUserInfo()->FindObject("AODObjectsConnectedToTree"));
     if (connectedList && (!strcmp(opt, "reconnect"))) {
+		
         // If connected use the connected list of objects
         if (fAODObjects != connectedList) {
            delete fAODObjects;
            fAODObjects = connectedList;
         }   
+        
+        friendL = tree->GetTree()->GetListOfFriends();
+		if (friendL) 
+		{
+		  // set the branch addresses
+		TIter next(fAODObjects);
```

Comment:
Looks like the indentation is wrong, may be due to use of tabs instead of spaces, can you fix it?

Reply:
This should be fixed now